app/src/main/java/com/nervousfish/nervousfish/activities/WaitActivity.java

```diff
@@ -33,6 +34,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("WaitActivity");
     private IServiceLocator serviceLocator;
+    private IDatabase database;
```

Comment:
Okey, but I hope you're aware the "preventing null pointer errors" doesn't hold here since you can't make the variable final (because it must be set in onCreate)